notebooks/tuning.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<p>Nice section. Like that you tie it all together.  Could again be nice to explain a bit what is AutoML if people are unfamiliar with the term/concept.  And also stress that you may not need all these parts to do something that is AutoML-like. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sktime/sktime-tutorial-pydata-global-2022/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>

Reply:
<p>yes good idea!</p>